components/TextFields/examples/TextFieldOutlinedExample.swift

```diff
@@ -192,6 +192,55 @@ final class TextFieldOutlinedSwiftExample: UIViewController {
                                                   metrics: nil,
                                                   views: views)
 
+    #if swift(>=4.0)
+    if #available(iOS 11.0, *) {
+            constraints += [NSLayoutConstraint(item: name,
+            attribute: .top,
+            relatedBy: .equal,
+            toItem: scrollView.contentLayoutGuide,
+            attribute: .top,
+            multiplier: 1,
+            constant: 20),
+            NSLayoutConstraint(item: message,
+            attribute: .bottom,
+            relatedBy: .equal,
+            toItem: scrollView.contentLayoutGuide,
+            attribute: .bottomMargin,
+            multiplier: 1,
+            constant: -20)]
+      } else {
```

Comment:
Formatting Nit: Did the "+=" make this format oddly?

Reply:
Oh, totally. Thank you.